suzieq/engines/pandas/network.py

```diff
@@ -179,7 +206,15 @@ def _find_addr_arp(self, addr: str, **kwargs) -> pd.DataFrame:
                      ifname=[oif])
 
             if not ifdf.empty:
-                if 'vlan' not in ifdf.type.unique():
+                netaddr = ip_address(row.ipAddress)
+                # The code below checks if the provided address belongs to
+                # the interface subnet. If it does, its a bridged address
+                # else its a routed address i.e. you use routing to reach it.
+                if not ifdf.ipAddressList.apply(
+                        lambda x, addr:
+                        False if not x.any() else
+                        any(addr in ip_network(y, strict=False) for y in x),
+                        args=(netaddr,)).any():
```

Comment:
This piece of code is not easy to follow, I would rename some of the variables:
```suggestion
                if not ifdf.ipAddressList.apply(
                        lambda addr_list, netaddr:
                        False if not addr_list.any() else
                        any(netaddr in ip_network(a, strict=False) for a in addr_list),
                        args=(netaddr,)).any():
```

Reply:
Fixed with diff var names